OPHD/States/MapViewState.h

```diff
@@ -214,6 +214,7 @@ class MapViewState : public Wrapper
 	void updateCommercial();
 	void updateMaintenance();
 	void updateMorale();
+	void notifyBirthsAndDeaths();
```

Comment:
I kind of like that you moved the notification code to a separate function from the `updateMorale` function.

Reply:
Thanks!